### ISSUE TITLE
[Avatar] 1.0.0-beta6 bugs and performance hazard

### ISSUE THREAD

**Issue by jackcbrown89** [react-native-training/react-native-elements#1459]
Just updated to `v1.0.0-beta6` and am now receiving a warning for my avatar component. After about five minutes of idle time, the CPU usage starts to climb from 0.5% CPU to 150% CPU

**Warning**
![simulator screen shot - iphone xs - 2018-10-12 at 11 03 09](https://user-images.githubusercontent.com/5186405/46877350-a81ae900-ce0e-11e8-880c-189543cac82f.png)

**My Code**
```
    <Avatar
          rounded
          size="xlarge"
          containerStyle={{ alignSelf: "center", marginBottom: 10}}
          source={{ uri: "https://pbs.twimg.com/profile_images/188302352/nasalogo_twitter_400x400.jpg" }}
    />
```

| software | version |
| -- | -- |
| react-native-elements | 1.0.0-beta6 |
| react-native | 0.57.1 |

**Comment by xavier-villelegier** [comment]
Mmh it seems like the warning comes from here, we need to fix that
https://github.com/react-native-training/react-native-elements/blob/e117365689a04119f2a796ba6e90e9f8f90474e3/src/avatar/Avatar.js#L214

For the CPU things, I don't think that it's strictly related to the `Avatar` component, as it is a simple view with some styling, even the fade in effect wouldn't make the CPU go to 150% 🤔

**Comment by iRoachie** [comment]
Closing this as fixed by #1507
